tests/test_mdm.py

```diff
@@ -0,0 +1,70 @@
+"""Methods for testing the MDM classifier module."""
+import geomstats.backend as gs
+import geomstats.tests
+from geomstats.geometry.spd_matrices import SPDMetricAffine
+from geomstats.learning.mdm import RiemannianMinimumDistanceToMeanClassifier
+
+E = gs.exp(1)
+
+
+class TestRiemannianMinimumDistanceToMeanClassifier(geomstats.tests.TestCase):
+    """Test of Riemannian MDM classifier."""
+
+    @geomstats.tests.np_only
+    def test_fit(self):
+        """Test the fit method."""
+        MDMEstimator = RiemannianMinimumDistanceToMeanClassifier(
+            SPDMetricAffine(n=2), point_type='matrix')
+
+        points_A = gs.array([[[E**2, 0], [0, 1]],
```

Comment:
no uppercase

Reply:
So just `e` for Euler's constant (is it somewhere in the lib already) ?

Reply:
Sorry, I hadnt realized if was a constant. If this case, `EULER = gs.exp(1.)` at the beginning of the file would be ideal actually. Then issue #403 will factorize the constants.

Reply:
Done.